spec/classes/foreman_proxy__plugin__dhcp__infoblox_spec.rb

```diff
@@ -51,6 +52,7 @@
             ':record_type: "host"',
             ':dns_view: "non-default"',
             ':network_view: "another-non-default"',
+            ':used_ips_search_type: "used"',
```

Comment:
I think you forgot to pass in a parameter for this, causing the tests to fail.

Reply:
Ups, right, thanks. Missed that!
Updated now.